docs/howto/write-scenario-tests-for-a-charm.md

```diff
@@ -1,45 +1,67 @@
 (write-scenario-tests-for-a-charm)=
-# How to write scenario tests for a charm
+# How to write unit tests for a charm
 
-First of all, install scenario:
+First of all, install the testing framework. To do this in a virtual environment
+while you are developing, you can do this with `pip` or any other package
+manager; for example:
```

Comment:
Since this is the first time we're mentioning the testing framework on the page, shall we call it the "Ops testing framework"? (Also included some very minor nits, since I'm making a suggestion)

```suggestion
First of all, install the Ops testing framework. To do this in a virtual environment
while you are developing, use `pip` or another package
manager. For example:
```

Reply:
Or "`ops` unit testing framework" for consistency with explanation/testing.md?